### DISCUSSION TITLE
Everything working fine, except live mode is not swapping

### DISCUSSION THREAD

**Discussion by unusualmutant** [hacksider/Deep-Live-Cam#352] [Q&A]
The installation went fine (Windows, RTX3090), and I can faceswap an image, but if I press "live" I just see myself in the preview and not a version with my face swapped for the selected image. Any ideas?

**Comment by De1stpromise** [comment]
> The installation went fine (Windows, RTX3090), and I can faceswap an image, but if I press "live" I just see myself in the preview and not a version with my face swapped for the selected image. Any ideas?

Hello please can you walk me through the installation process on my computer please? It's been quite tricky getting it set up.if you don't mind seeing me through the installation process.

**Reply by unusualmutant** [reply]
I actually just followed all the steps exactly as described, and made sure that my terminal windows were running as administrator (very important). When I didn't do that it failed.

**Comment by hacksider** [comment]
If you're using 3090, just use this package :) 
https://github.com/hacksider/Deep-Live-Cam/releases/tag/cuda_deep-live-cam

**Reply by unusualmutant** [reply]
You're a star, thanks!

**Reply by unusualmutant** [reply]
Same result though. I select a source picture, click live, and all I see is myself on the webcam :/
